torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1965,6 +1946,10 @@ void ONNXShapeTypeInference(
     Node* n,
     const ParamMap& params_dict,
     int opset_version) {
+  std::unordered_map<std::string, std::string> outputs_map;
+  auto generated_shape_data = ConstantValueMap::GetGeneratedShape();
```

Comment:
Ditto on use one consistent name: either add `Data` to the function name, or remove it from the variable name.